### PR TITLE
Feature card

### PR DESCRIPTION
**Pull request by Bhakti199**
here I added different types of card-

1. vertical card
2. card with badge
3. horizontal card
4. text only card
5. card with dismiss
6. card with text overlay

### CODE REVIEW

**Review by Raghav888 — APPROVED** [review]

File: button/button.css [hunk 1]
```diff
@@ -6,53 +6,68 @@
...
 .btn-container {
   display: flex;
-  flex-direction: column;
+  flex-wrap: wrap;
   align-items: center;
   gap: 2rem;
-  height: 16rem;
-  margin-bottom: 1rem;
+  margin-top: 2rem;
+  margin-bottom: 2rem;
```
Comment (fahd23) [inline button/button.css]: here you can use a margin shortcut, if you want

File: button/button.css [hunk 2]
```diff
@@ -6,53 +6,68 @@
...
   border: none;
   color: var(--dark-black);
   border: none;
+  border-radius: 3px;
   font-size: 1.25rem;
 }
-
+.secondary-btn-md {
+  border: none;
+  border: none;
```
Comment (fahd23) [inline button/button.css]: Repeated

File: button/button.css [hunk 3]
```diff
@@ -6,53 +6,68 @@
...
+}
+.primary-btn-md-two {
+  background-color: var(--medium-grey);
+}
+.primary-btn-md-one,
+.primary-btn-md-two,
+.secondary-btn-md,
+.btn-with-icon-md {
+  width: 16rem;
+  padding: 11.2px 2rem;
```
Comment (fahd23) [inline button/button.css:58]: use rem or px here, I don't think it is a good practice

**Review by fahd23 — CHANGES_REQUESTED** [review]

File: button/button.css (hunk 3 above)
Comment (Bhakti199) [reply button/button.css:58]: its too small value thats why I use px.

File: button/button.css [hunk 4]
```diff
@@ -6,53 +6,66 @@
 }
 .btn-container {
   display: flex;
-  flex-direction: column;
+  flex-wrap: wrap;
   align-items: center;
   gap: 2rem;
-  height: 16rem;
-  margin-bottom: 1rem;
+  margin: 2rem 0 2rem 0;
```
Comment (fahd23) [inline button/button.css:12]: you can use margin shorthand here

**Review by fahd23 — APPROVED** [review]
Good to merge

**Pull request merged**